src/lib/CheckoutWithCard.ts

```diff
@@ -114,6 +114,10 @@ export function createCheckoutWithCardMessageHandler({
         break;
 
       case 'openModalWithUrl':
+        modal = new Modal(undefined, {
```

Comment:
Looking at this code previously this used to reuse a single, existing modal. 

`open()` calls `document.body.appendChild(this.main)`.
`close()` calls `this.main.remove()

I'm playtesting this now and this seems to allow opening the modal multiple times as long as `close()` was called before another `open()`. Was there a reason to switch to instantiating a new Modal each time?

Reply:
Have you tried changing the content inside the modal? Maybe that's what causes the issue. I can guarantee that something occurred that caused open to not work, and doing this fixed it. I can take a closer look.